xml/smb_native_lm.xml

```diff
@@ -41,5 +41,21 @@
     <param pos="0" name="service.vendor" value="Samba"/>
     <param pos="0" name="service.product" value="Samba"/>
     <param pos="1" name="service.version"/>
+    <param pos="0" name="service.certainty" value="1.00"/>
   </fingerprint>
+
+  <!-- Windows -->
+  <fingerprint pattern="^(Windows [\W*\w*]*)$">
+    <description>Windows</description>
+    <example service.product="Windows Server 2008 R2 Enterprise 6.1">Windows Server 2008 R2 Enterprise 6.1</example>
```

Comment:
Well sure, but now you are missing an OS fingerprint.  

In general, IMO, for every "banner" that recog is capable of matching against, we should assert as much fingerprint information as possible.  The LM and Native OS strings here should allow to assert both an operating system and a service fingerprint.